doc/stdlib/linalg.rst

```diff
@@ -101,6 +101,10 @@ Linear algebra functions in Julia are largely implemented by calling functions f
 
    Compute a factorization of a positive definite matrix ``A`` such that ``A=L*Diagonal(d)*L'`` where ``L`` is a unit lower triangular matrix and ``d`` is a vector with non-negative elements.
 
+.. function:: ldltfact(A) -> CHOLMOD.Factor
+
+   Compute the LDLt factorization of a sparse symmetric or Hermitian matrix ``A``. A fill-reducing permutation is used.  The main application of this type is to solve systems of equations with ``\``, but also the methods ``diag``, ``det``, ``logdet`` are defined. The function calls the C library CHOLMOD and many other functions from the library are wrapped but not exported.
```

Comment:
Yes. Was about to write that I think I've solved the first part, but that improving the documentation remains.

I like the the idea of overloading `factorize` with e.g. `Val{:Cholesky}`, `Val{:LDLt}`, `Val{:LU}` instead of `cholfact`, `ldltfact`, etc, but I'm still not really happy with the name `Val`.